frontend/cypress.config.ts

```diff
@@ -0,0 +1,12 @@
+import { defineConfig } from "cypress";
+
+export default defineConfig({
+  e2e: {
+    setupNodeEvents(on, config) {
+      // implement node event listeners here
+    },
+  },
+  env: {
+    CYPRESS_baseUrl: 'http://localhost:3000',
```

Comment:
this will work on local dev...but how will it hold up moving through the environments?